gui/default/assets/lang/lang-en.json

```diff
@@ -134,6 +134,7 @@
    "Folder Label": "Folder Label",
    "Folder Path": "Folder Path",
    "Folder Type": "Folder Type",
+   "Folder type \"{%receiveEncrypted%}\" cannot be changed after adding the folder. You need to remove the folder, delete or decrypt the data on disk, and add the folder again.": "Folder type \"{{receiveEncrypted}}\" cannot be changed after adding the folder. You need to remove the folder, delete or decrypt the data on disk, and add the folder again.",
```

Comment:
Could we not just say "Receive Encrypted" here without the magic token? It will be translated as part of the string as a whole?

Reply:
I thought it was a nice touch to ensure consistent translations being used, given this is a "label/name", not just a normal language phrase. I.e. preventing the possibility that the string "Receive Encrypted" alone gets translated to something different than in the context of this sentence.

Reply:
Okay, let's leave it as-is.